packages/kbn-flot-charts/lib/jquery_flot.js

```diff
@@ -1,3168 +1,3777 @@
-/* JavaScript plotting library for jQuery, version 0.8.3.
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+/* Javascript plotting library for jQuery, version 0.9.0-alpha.
 
-Copyright (c) 2007-2014 IOLA and Ole Laursen.
+Copyright (c) 2007-2013 IOLA and Ole Laursen.
```

Comment:
nit, let's keep the years the same.

```suggestion
Copyright (c) 2007-2014 IOLA and Ole Laursen.
```